swath_projector/earthdata_varinfo_config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "Identification": "Swath Projector VarInfo configuration",
+  "Version": 3,
+  "CollectionShortNamePath": [
+    "ShortName"
+  ],
+  "Mission": {
+    "VNP10": "VIIRS"
+  },
+  "MetadataOverrides": [
+    {
+      "Applicability": {
+        "Mission": "VIIRS",
+        "ShortNamePath": "VNP10",
+        "VariablePattern": "/SnowData/.*"
+      },
+      "Attributes": [
+        {
+          "Name": "coordinates",
+          "Value": "/GeolocationData/latitude, /GeolocationData/longitude"
+        }
+      ],
+			"_Description": "VNP10 SnowData variables have incorrect relative paths for coordinates."
+    }
```

Comment:
This got tabbified somehow.

Reply:
Huh - weird. I thought my editor was set up to automatically convert tabs to spaces. Ah well, I updated that [here](https://github.com/nasa/harmony-swath-projector/pull/18/commits/2cd831067567814c0997e6c4bfb9820cff502d67).